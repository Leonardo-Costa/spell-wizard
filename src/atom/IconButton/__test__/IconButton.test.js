import React from 'react';
import renderer from 'react-test-renderer';

import {IconButton} from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<IconButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });