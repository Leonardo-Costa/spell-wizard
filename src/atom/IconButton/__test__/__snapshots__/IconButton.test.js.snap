// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "opacity": 1,
        "padding": 2,
      }
    }
  >
    <Image
      style={
        Object {
          "height": 24,
          "width": 24,
        }
      }
    />
  </View>
</View>
`;
